proto/minder/v1/minder.proto

```diff
@@ -2317,7 +2317,14 @@ message UpdateProjectRequest {
     ];
 
     // description is the description of the project to update.
-    string description = 3;
+    string description = 3 [
+        (buf.validate.field).string = {
+            pattern: "[[:space:][:punct:][:word:]]*",
+            min_len: 1,
```

Comment:
If `min_len` is 0, you don't need the `ignore` annotation.

Reply:
Fixed it 👍